src/Microsoft.Azure.SignalR/HubHost/ServiceHubDispatcher.cs

```diff
@@ -26,6 +27,7 @@ internal class ServiceHubDispatcher<THub> : IConnectionFactory where THub : Hub
         private readonly IClientConnectionFactory _clientConnectionFactory;
         private readonly string _userId;
         private static readonly string Name = $"ServiceHubDispatcher<{typeof(THub).FullName}>";
+        private static Dictionary<string, string> CustomHeader = ProductInfo.ToHeader();
```

Comment:
>ToHeader [](start = 77, length = 8)

It is possible that we might have other items in the header, so I think it is more appropriate for `ProductInfo` to just return UA information instead of the whole header

Reply:
The header inits when we new HttpConnectionOptions(), so should be safe to new header here.